### DISCUSSION TITLE
Hacktoberfest contributions

### DISCUSSION THREAD

**Discussion by Connor-Knabe** [martomi/chiadog#300] [Ideas]
Hey @martomi any chance you can add hacktoberfest as a topic for this repo? I would love to submit some pull requests for Hacktoberfest to make some improvements to your program like more configuration options and maybe work on some of the issues reported too.

Thanks for making this tool it, helps me sleep easy at night knowing the farm is working without issues!

**Comment by martomi** [comment]
Hi @Connor-Knabe, great idea! I just added the topic, looking forward to your PRs :-)

**Reply by Connor-Knabe** [reply]
@martomi awesome! I have a quick one in and have 3 other ones I'm working on (plot notifications which are configurable, ifttt, and pushcut notification support) but need to do a bit more testing to confirm they are working before submitting the PRs

**Comment by Connor-Knabe** [comment]
@martomi I submitted those other pull requests. If you’re not comfortable to merge them yet could you tag them with “hacktoberfest-accepted” label please? 

I’d be happy to implement any suggestions you have about the code. Been running the plot in increasing branch for a few days now and it’s been useful (randomly my usb HDDs will disconnect but then reconnect it’s good to see when they come back online)

Thanks!

**Reply by martomi** [reply]
Hi @Connor-Knabe , I've been running somewhat low on time recently and didn't get chance to look into them yet. Will do so in the next couple of days but certainly before end of the month!
